cluster-autoscaler/cloudprovider/clusterapi/clusterapi_utils_test.go

```diff
@@ -505,23 +525,43 @@ func TestParseMemoryCapacity(t *testing.T) {
 		expectedQuantity: zeroQuantity.DeepCopy(),
 		expectedError:    false,
 	}, {
-		description:      "bad quantity",
+		description:      "upstream bad quantity",
 		annotations:      map[string]string{memoryKey: "not-a-quantity"},
 		expectedQuantity: zeroQuantity.DeepCopy(),
 		expectedError:    true,
 	}, {
-		description:      "quantity as with no unit type",
+		description:      "upstream quantity as with no unit type",
 		annotations:      map[string]string{memoryKey: "1024"},
 		expectedQuantity: *resource.NewQuantity(1024*units.MiB, resource.DecimalSI),
```

Comment:
@elmiko I've changed the two cases below this, but I'm not quite sure what way this one should change now - it's failing with `expected 1073741824, got 1024`

Reply:
i'm not sure that the value `"1024"` converts to `MiB`, it might be `GiB`. look up the docs for `resource.Quantity` to read about how it converts a value with no unit after it.

try using `resource.MustParse("1024")` instead of the `NewQuantity`, maybe that would fix it.

Reply:
I think you're right - this change fixed it